README_DEVELOPERS.md

```diff
@@ -19,7 +19,7 @@ The `cd-service` takes the URL of the repository to watch from the environment v
 
 You need a `builder` image that is tagged as `latest` to build services locally.
 There's 2 ways to get an image:
-* `IMAGE_TAG=latest make -C infrastructure/docker/builder build`
+* `make builder` (recommended for independant contributors)
```

Comment:
I would argue this is recommended for everyone

Reply:
Simplified accordingly.